pages/common/mysqld.md

```diff
@@ -0,0 +1,19 @@
+# mysqld
+
+> The MySQL Server.
+
+- Display error log messages:
+
+`mysqld --console`
+
+- Log connections and queries to a file:
+
+`mysqld --log={{file}}`
+
+- Log errors and start messages a file:
```

Comment:
But isn't one `--log` and the other `--log-error`, each doing different (i.e. mutually exclusive) things? Please correct me if I'm wrong, but otherwise, I don't think there's much to gain by using a single logging example, considering that the page only has 4 examples at the moment.